src/ConnectedMode.UnitTests/Binding/NonRoslynBindingConfigProviderTests.cs

```diff
@@ -37,26 +37,18 @@ public class NonRoslynBindingConfigProviderTests
     {
         private static readonly Language[] SupportedLanguages = { Language.C, Language.Cpp };
 
-        [TestMethod]
-        public void Ctor_NullSupportedLanguages_ArgumentNullException()
```

Comment:
The supported languages are no longer set in the public constructor, so I removed the null check